package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Tizen Advanced UI",
   "version": "1.0.25",
   "scripts": {
+    "build": "npx grunt build",
```

Comment:
grunt build
will not be enough?
in 1.1 it is done this way
https://github.com/Samsung/TAU/blob/tau_1.1/package.json

Reply:
Ok I will check this.

Reply:
You're right. I also updated README.md accordingly.